index.html

```diff
@@ -74,6 +74,7 @@
           <h2>Heading</h2>
           <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
           <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <!-- Fixed Image Gallery -->
           <i class="fa fa-bookmark" aria-hidden="true"></i> <!-- Allows user to favorite poem -->
```

Comment:
In addition to bookmarking the poem we should also add social icons so that the use may share the poem via social media: Facebook, Twitter, and Pinterest.